BunnyHopMaster/Assets/Scripts/Player/PlayerProgress.cs

```diff
@@ -9,6 +10,7 @@ public class PlayerProgress : MonoBehaviour
     public PlayerUI playerUI;
     public bool didWin = false;
     public float minimumY = 0;
+    public string last_Scene_name;
```

Comment:
I think we should stick to one naming convention. Generally, C# is using camel case naming for variables, so I think we should name it `lastSceneName` here.